packages/extension/src/background/index.ts

```diff
@@ -113,7 +114,8 @@ const safeMessages: MessageType["type"][] = [
   "ALPH_EXECUTE_TRANSACTION_RES",
   "TRANSACTION_SUBMITTED",
   "TRANSACTION_FAILED",
-  "ALPH_SIGN_MESSAGE_RES",
+  "ALPH_SIGN_RES",
```

Comment:
My initial thought was to share some message types, but now it might look confusing.

In the latest code, the message types for sign message are:

`ALPH_SIGN_MESSAGE`
`ALPH_SIGN_MESSAGE_RES`
`ALPH_SIGN_MESSAGE_SUCCESS`, renamed from `SIGNATURE_SUCCESS`
`ALPH_SIGN_MESSAGE_FAILURE`, renamed from `SIGNATURE_FAILURE`
`SIGN_MESSAGE`, renamed from `SIGN`

the message types for sign unsigned tx are:

`ALPH_SIGN_UNSIGNED_TX`
`ALPH_SIGN_UNSIGNED_TX_RES`
`ALPH_SIGN_UNSIGNED_TX_SUCCESS`
`ALPH_SIGN_UNSIGNED_TX_FAILURE`
`SIGN_UNSIGNED_TX`